package com.wsu.crimsonconnect.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
public class SpaController {
    @RequestMapping(value = {"/{path:[^\\.]*}", "/**/{path:^(?!api$).*$}"})
    public String redirect() {
        return "forward:/index.html";
    }
}
